docs/reference/terminal49/terminal49.v1.json

```diff
@@ -5164,7 +5331,106 @@
           }
         },
         "operationId": "get-vessels-id",
-        "description": "Returns a vessel by it's given identifier"
+        "description": "Returns a vessel by the given identifier\n\n<!-- theme: warning -->\n> #### BETA notice\n> The `show_positions` query paramaters and `positions` attribute are currently in beta and subject to revision.",
+        "parameters": [
+          {
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "in": "query",
+            "name": "show_positions[from_timestamp]",
+            "description": "Starting timestamp for showing vessel positions"
+          },
+          {
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "in": "query",
+            "name": "show_positions[to_timestamp]",
+            "description": "Ending timestamp for showing vessel positions"
+          }
+        ]
+      }
+    },
+    "/vessels/{id}/future_positions_with_coordinates": {
```

Comment:
@mattyturner 

We should remove this – this is a _very_ temporary endpoint; awaiting a proposal from Bart.